Azure/sonic-buildimage
# [Ingrasys] Enable SSD fstrim service and timer for Broadcom platforms

**- What I did**
Add fstrim service and timer for broadcom platforms (s8810/s8900-54xc/s8900-64xc/s9100/s9200).
The fstrim service run once a week.

**- How I did it**
Add fstrim.service and fstrim.timer in systemd

**- How to verify it**
Execute "systemctl list-timers" to verify next execution time for fstrim.service

**- Description for the changelog**
[Ingrasys] add fstrim service and timer for broadcom platforms

**- A picture of a cute animal (not mandatory but encouraged)**
